### ISSUE TITLE
realizations missing fields

### ISSUE THREAD

**Issue by kohlhase** [KWARC/sTeX#365]
In `smglom/linear-algebra/source/mod/vector-space.en.tex` I have 
<img width="601" alt="Screenshot 2022-09-30 at 15 01 10" src="https://user-images.githubusercontent.com/4037538/193278133-1b148cef-d5d5-4b7f-bf43-36498147617a.png">
and indeed there are some fields missing, but in this case, I do not want to do all the (trivial) assignments. Is there a way to be more convenient here?

**Comment by Jazzpirate** [comment]
well, the actual targets of the assignments are only checked by MMT, so from the point of view of sTeX you can easily do
\assign{foo}{I have a marvelous proof which this .tex file is too narrow to contain} - but a realization requires assigning all fields in principle, so I think it useful to have latex insist on it and have MMT complain ;)

**Comment by Jazzpirate** [comment]
Ah, for the special case where you want to assign the *identity* on a whole module/mathstructure, there is the abbreviation `\donotclone{module-name}`, which automatically assign everything in `module-name` to itself, so modularity helps/is encouraged

**Comment by kohlhase** [comment]
Ah, that is what I was hoping for, does that take a CSL as the argument?

**Comment by Jazzpirate** [comment]
No, but I'd be surprised if the situation occurs too often that you have several (>2) modules you want to assign to the identity...
